src/core/Utils.js

```diff
@@ -527,12 +527,18 @@ function findDOMElement (element) {
 /**
  * Find one or more DOM elements.
  *
- * @param {string} selector
+ * @param {string} element
  * @return {Array|null}
  */
-function findAllDOMElements (selector) {
-  const elements = [].slice.call(document.querySelectorAll(selector))
-  return elements.length > 0 ? elements : null
+function findAllDOMElements (element) {
+  if (typeof element === 'string') {
+    const elements = [].slice.call(document.querySelectorAll(element))
+    return elements.length > 0 ? elements : null
+  }
+
+  if (typeof element === 'object' && isDOMElement(element)) {
+    return element
```

Comment:
If we return an array with a single item here (`return [element]`), the code that checks if the result is an array when adding and removing events can be simplified to always assume an array.

Reply:
Sorry, I missed that. Done!